table.cc

```diff
@@ -2115,7 +2114,7 @@ void
 table::enable_auto_compaction() {
     // XXX: unmute backlog. turn table backlog back on.
     //      see table::disable_auto_compaction() notes.
-    _compaction_disabled_by_user = false;
+    _config.enable_auto_compaction = true;
```

Comment:
Won't this affect all tables now, as `_config.enable_auto_compaction` is a live value.

Reply:
yes, you're right.
we need per table state in this case.

Reply:
I think we can just retain the existing field in `table` to act as a local override, and we `&&` the global and local config to compute the value relevant for the table.